meson.build

```diff
@@ -498,6 +498,7 @@ foreach package : subprojects
       endif
     endif
   else
+    set_variable(package_name, declare_dependency())
```

Comment:
Sorry, but I didn't get why this is needed, could you explain for me?

Reply:
At first, some meson.builds were complaining about unknown static_libs, so this was just a shortcut (no preprocessor symbol is defined, so `HAVE_*` won't mislead, etc.).